install_sct

```diff
@@ -643,5 +646,5 @@ $DISPLAY_UPDATE_PATH"
 source $RC_FILE_PATH"
   fi
 else
-  die "Installation validation Failed!"
+  echo "Installation validation Failed!"
```

Comment:
`ivadomed` is in pypi now so this can go back to how it was.

I want to think about the part of `sct_check_dependencies` that tries to parse requirements.txt. It's only maybe 99.9% reliable, but fixing that is not part of this PR.

Reply:
fixed in ff92d44032e907aff8bf76d7bf99b9f1c769cce4